docs/iam-user-service.md

```diff
@@ -20,7 +20,7 @@ To create an IAM user:
 
     A Lambda function will trigger to create the IAM user, generate an access key, and store it in the SSM Parameter Store. This function also runs hourly to rotate keys as needed and ensure DynamoDB table entries align with actual IAM account users, removing any discrepancies.
 
-      ![iam-user](images/iam-user-service/iam-users.png)
+    ![iam-user](images/iam-user-service/iam-users.png)
```

Comment:
Hi @MaxWardle  can you please create a folder called images, upload the images there and then the URL should be: 

[Image example](../../images/imagename.png)

Reply:
Hey @Pilargit12, this folder exists [here](https://github.com/bcgov/public-cloud-techdocs/tree/dev/docs/images) already, this change was a formatting issue. Markdown doesn't like that space I removed, but the image should render now.